zokrates_core/src/ir/smtlib2.rs

```diff
@@ -0,0 +1,141 @@
+use num_bigint::BigUint;
+use std::collections::BTreeSet;
+
+use super::*;
+use zokrates_field::Field;
+
+use super::expression::LinComb;
+use super::expression::QuadComb;
+use super::visitor::*;
+
```

Comment:
@Schaeff not sure if you have a convention for the order of these /\

Reply:
No